### PR TITLE
Fix open machete file action in the case missing machete file

### PR DESCRIPTION
**Pull request by mkondratek**
Steps to reproduce:

1. have git machete tab closed 
2. remove machete file
3. call `open machete file` from the find action dialog

result:
error - machete file not found
notification - branch layout discovered

expected: 
tbd

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java [hunk 1]
```diff
@@ -56,8 +56,6 @@ public void actionPerformed(AnActionEvent anActionEvent) {
             /* message */ getString("action.GitMachete.OpenMacheteFileAction.notification.title.cannot-open"))))
         .getOrNull();
 
-    getGraphTable(anActionEvent).queueRepositoryUpdateAndModelRefresh();
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java:59]: Wow WTF was the purpose of this... can you dig out any clue from the original PR that added this line?

Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java:59]: I searched the git history and looked at https://github.com/VirtusLab/git-machete-intellij-plugin/pull/272 but I cannot see anything that could explain that

**Review by PawelLipski — APPROVED** [review]

**Comment by mkondratek** [comment]
I added the notification action so one can easily resolve the problem described in error:

image

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java [hunk 2]
```diff
@@ -56,13 +60,18 @@ public void actionPerformed(AnActionEvent anActionEvent) {
...
-
     if (macheteFile == null) {
-      VcsNotifier.getInstance(project).notifyError(/* displayId */ null,
+      val errorWithDiscover = VcsNotifier.STANDARD_NOTIFICATION.createNotification(
           /* title */ getString("action.GitMachete.OpenMacheteFileAction.notification.title.machete-file-not-found"),
           /* message */ getString("action.GitMachete.OpenMacheteFileAction.notification.message.machete-file-not-found")
-              .fmt(gitDir.getPath()));
+              .fmt(gitDir.getPath()),
+          NotificationType.ERROR);
+      errorWithDiscover.addAction(NotificationAction
```
Comment (mkondratek) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java:69]: Note there is no `expire()`. It's hard to track the notification status and all the cases that could make it expired. On the other hand the action is simply a call to discover action - one see the suggested dialog first, must accept it. So even though the action will remain available as long as the notification lives there is no big deal imho. wdyt @PawelLipski ?

Comment (PawelLipski) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java:69]: Makes sense... maybe pls put the above musings into a code comment!

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**